hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/hevm/HederaEvmTransactionTest.java

```diff
@@ -52,4 +52,11 @@ void computesOfferedGasCostWithOverflow() {
         final var subject = TestHelpers.wellKnownRelayedHapiCallWithGasLimit(Long.MAX_VALUE);
         assertEquals(Long.MAX_VALUE, subject.offeredGasCost());
     }
+
+    @Test
+    void validateContractCallState() {
+        final var subject = TestHelpers.wellKnownHapiCall();
+        assertTrue(subject.isContractCall());
+        assertFalse(subject.isException());
+    }
```

Comment:
I guess it's ok to _not_ test when it's _not_ a contract call, or _is_ an exception?